packages/audioplayers/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: audioplayers
 description: A Flutter plugin to play multiple audio files simultaneously
-version: 0.20.1
+version: 0.21.0
```

we can leave this as is, I will bump everything later (probably will add melos for repo-wide management)
```suggestion
version: 0.20.1
```